test/topics/dlq.test.js

```diff
@@ -44,14 +44,8 @@ describe('Topics - Systemic Azure Bus API', () => {
 	});
 
 	it('DLQ peek - should contain one message', () => new Promise(async resolve => {
-		const BULLETS = 1;
+		const BULLETS = 2;
```

Comment:
Because in the past `attack` method, every publish was made depending of an array created based on those bullets. Eg, if we had 1 bullet, it was created an array like this `[0, 1]` on which we iterated in order to make those publish. But now we have a kinda different recursive method that depending of the amount (bullets) it will be iterating over itself until the amount will be 0. Just to sum up, before, the iteration was over `[0, 1]`, so it made the publication twice, and in the new method I had to increment to 2 in order to make the same amount of iterations. And, in fact, I think now the name var `bullets` has a little more sense.

Reply:
👏 👏 👏